dsls/sql/src/main/java/org/apache/beam/dsls/sql/interpreter/operator/arithmetic/BeamSqlArithmeticExpression.java

```diff
@@ -80,12 +80,10 @@ public BeamSqlArithmeticExpression(List<BeamSqlExpression> operands) {
 
   private double getDouble(BeamSqlRow inputRecord, BeamSqlExpression op) {
     Object raw = op.evaluate(inputRecord).getValue();
-    Double ret = null;
     if (SqlTypeName.NUMERIC_TYPES.contains(op.getOutputType())) {
-      ret = ((Number) raw).doubleValue();
+      return ((Number) raw).doubleValue();
     }
-
-    return ret;
+    return 0d;
```

Comment:
You have a point with the return, I also hesitated, probably IllegalStateException makes more sense, I am going to change, also I fixed the case of division that didn't validate division by zero, I am going to add this here too. Thanks for the review.